tests/conftest.py

```diff
@@ -31,6 +32,38 @@ def json(self):
         return json.loads(self.text)
 
 
+def fixed_get_environ():
```

Comment:
The `WSGIMiddleware` and test client were unfortunately not compatible (see https://github.com/abersheeran/a2wsgi/issues/21, https://github.com/pallets/werkzeug/issues/2347). The issues are fixed, but I still had to patch the test client since the fixes are not released and I don't want to restrict the versions too much.